### PR TITLE
Language chooser

### PR DESCRIPTION
**Pull request by Alir3z4**
@AmirSavand you can add country flags to this.
Just make the country flags file like:

ru.png
en.png
hi.png

for now we have only 3 languages.

### CODE REVIEW

File: kloud51/templates/menu/language_chooser.html [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+{# TODO: Let's have the Language (country) flags as well. #}
+
+<li class="dropdown">
+  <a class="dropdown-toggle" data-toggle="dropdown" href="#" id="Language" aria-expanded="false">
+    {{ current_language }} <span class="caret"></span>
+  </a>
+  <ul class="dropdown-menu">
+    {% for language in languages %}
+      <li class="lang{% if current_language == language.0 %} active{% endif %}">
+        <a href="{% page_language_url language.0 %}" title="{% trans "Change to language:" %} {{ language.1 }}">{{ language.1 }}</a>
```
Comment (Alir3z4) [inline kloud51/templates/menu/language_chooser.html:11]: @AmirSavand this is where you'd include the language field flag.

Comment (AmirSavand) [reply kloud51/templates/menu/language_chooser.html:11]: ok

**Review by AmirSavand — APPROVED** [review]

**Comment by Alir3z4** [comment]
We're in

**Comment by AmirSavand** [comment]
Nice.

**Pull request merged**